network/src/protocols/support_protocols.rs

```diff
@@ -91,18 +93,23 @@ impl SupportProtocols {
         // we didn't invoke MetaBuilder#support_versions fn for these protocols (Ping/Discovery/Identify/Feeler/DisconnectMessage)
         // in previous code, so the default 0.0.1 value is used ( https://github.com/nervosnetwork/tentacle/blob/master/src/builder.rs#L312 )
         // have to keep 0.0.1 for compatibility...
-        match self {
-            SupportProtocols::Ping => vec!["0.0.1".to_owned(), "2".to_owned()],
-            SupportProtocols::Discovery => vec!["0.0.1".to_owned(), "2".to_owned()],
-            SupportProtocols::Identify => vec!["0.0.1".to_owned(), "2".to_owned()],
-            SupportProtocols::Feeler => vec!["0.0.1".to_owned(), "2".to_owned()],
-            SupportProtocols::DisconnectMessage => vec!["0.0.1".to_owned(), "2".to_owned()],
-            SupportProtocols::Sync => vec!["1".to_owned(), "2".to_owned()],
+        let mut versions = match self {
+            SupportProtocols::Ping => vec!["0.0.1".to_owned(), LASTEST_VERSION.to_owned()],
+            SupportProtocols::Discovery => vec!["0.0.1".to_owned(), LASTEST_VERSION.to_owned()],
+            SupportProtocols::Identify => vec!["0.0.1".to_owned(), LASTEST_VERSION.to_owned()],
+            SupportProtocols::Feeler => vec!["0.0.1".to_owned(), LASTEST_VERSION.to_owned()],
+            SupportProtocols::DisconnectMessage => {
+                vec!["0.0.1".to_owned(), LASTEST_VERSION.to_owned()]
+            }
+            SupportProtocols::Sync => vec!["1".to_owned(), LASTEST_VERSION.to_owned()],
             SupportProtocols::Relay => vec!["1".to_owned()],
-            SupportProtocols::Time => vec!["1".to_owned(), "2".to_owned()],
-            SupportProtocols::Alert => vec!["1".to_owned(), "2".to_owned()],
-            SupportProtocols::RelayV2 => vec!["2".to_owned()],
-        }
+            SupportProtocols::Time => vec!["1".to_owned(), LASTEST_VERSION.to_owned()],
+            SupportProtocols::Alert => vec!["1".to_owned(), LASTEST_VERSION.to_owned()],
+            SupportProtocols::RelayV2 => vec![LASTEST_VERSION.to_owned()],
+        };
+        // make sure version list is sorted
+        versions.sort();
```

Comment:
we should not use `sort` here since it's string vec:
```
["0.0.1", "2", "10"].sort => ["0.0.1", "10", "2"]
```

Reply:
I'll change it to add a comment, the order here will affect the handshake behavior